modules/standard/Map.chpl

```diff
@@ -109,6 +102,13 @@ module Map {
       this.parSafe = parSafe;
     }
 
+    proc init(type keyType, type valType, param parSafe=false)
```

Comment:
While we're here, do we want to add a compiler warning for `init` and `init=` if the value type is an `owned` class? Since we've more or less confirmed that owned classes won't work for now.